omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -91,3 +91,10 @@
 </form>
 
 <div id="patient-search-results"></div>
+
```

Comment:
Is this on top of the results table? I would think this button should be at the top of the results table